### DISCUSSION TITLE
Splindle

### DISCUSSION THREAD

**Discussion by BeAutomacaoPlax** [winder/Universal-G-Code-Sender#1835] [Help]
Como faço para selecionar quando acionar o splindle ?

**Comment by breiler** [comment]
Not sure if I follow. If you have support for starting and controlling the spindle speed from your controller you can usually generate gcode to include such commands (M3, M4 and M5):
https://linuxcnc.org/docs/html/gcode/m-code.html#mcode:m3-m4-m5
